### ISSUE TITLE
Untangle tests

### ISSUE THREAD

**Issue by abeforgit** [abeforgit/RestoWeb#23]
Tests added in #20 should be separated into their own functions. this allows the test runner to keep running the rest of the tests when one fails. The test method name should be what describes the test and can therefore be ridiculously long

**Comment by Robbe7730** [comment]
This is true, however I wrote the tests to make sure the endpoints worked without keeping 100 curl-commands and to make sure the endpoints made sense. It is indeed a dirty hack to keep them all in one function, but the problem with splitting them up is that one test may rely on a previous test, e.g. if we can't create a Resto, we can't create a Menu linked to that resto either. So keeping it all in one function and thus one flow was the easiest

**Comment by abeforgit** [comment]
Yeah I realized that once I tried to split them up. Totally makes sense since we're so pressed for time. Ill mark this as low priority.
